build.sbt

```diff
@@ -336,13 +336,6 @@ lazy val enumeratumCirce = crossProject(JSPlatform, JVMPlatform)
       )
     }
   )
-  .jsSettings(libraryDependencies ++= (CrossVersion.partialVersion(scalaVersion.value) match {
```

Comment:
I realised after committing that the `libraryDependencies` may still be needed? I was working off reverting the part of #241 that added `not-java-time`, but may have been too aggressive...

Reply:
I think this is fine; the only goal of this was to add `not-java-time` to the JS dependencies.